src/main/target/ANYFCM7/target.h

```diff
@@ -31,20 +31,22 @@
 #define BEEPER_PIN PB2 // Unused pin, can be mapped to elsewhere
 #define BEEPER_INVERTED
 
-#define MPU6000_CS_PIN        PA4
-#define MPU6000_SPI_INSTANCE  SPI1
+#define GYRO_1_CS_PIN         PA4
+#define GYRO_1_SPI_INSTANCE   SPI1
 
 #define USE_ACC
 #define USE_ACC_SPI_MPU6000
-#define ACC_MPU6000_ALIGN CW270_DEG
+#define ACC_1_ALIGN       CW270_DEG
 
 #define USE_GYRO
 #define USE_GYRO_SPI_MPU6000
-#define GYRO_MPU6000_ALIGN CW270_DEG
+#define GYRO_1_ALIGN       CW270_DEG
 
 // MPU6000 interrupts
 #define USE_MPU_DATA_READY_SIGNAL
-#define MPU_INT_EXTI PC4
+#define USE_EXTI
+#define USE_GYRO_EXTI
+#define GYRO_1_EXTI_PINPC4
```

Comment:
Is this a typo? There's a whole bunch of `GYRO_1_EXTI_PINPxx` that look like search / replace gone bad.